planner/core/memtable_predicate_extractor.go

```diff
@@ -541,6 +541,24 @@ func (helper extractHelper) convertToTime(t int64) time.Time {
 	return time.Unix(0, t)
 }
 
+func (helper extractHelper) extractRoleBoolSliceFromUint64Slice(uint64Slice []uint64) []bool {
```

Comment:
The function name is too long.

Reply:
change to `convertToBoolSlice` according to previous func `convertToTime`.